Scenes/ContentManager/Custom_Editors/TacticalMapEditor/Scripts/TacticalMapEditorTileBrush.gd

```diff
@@ -23,3 +33,28 @@ func set_selected(is_selected: bool) -> void:
 		modulate = Color(0.227, 0.635, 0.757)
 	else:
 		modulate = Color(1,1,1)
+
+
+# Set the label text and adjust the size of the label and the tile sprite accordingly
+func set_label(text: String):
```

Comment:
I see that in this function we use a lot of `64`. I think that we should set it to some const. Like: `const DEFAULT_WIDTH = 64` or something similar.

Reply:
Implemented your suggestion